corehq/apps/ota/views.py

```diff
@@ -377,11 +375,25 @@ def form_valid(self):
 
 @login_or_digest_or_basic_or_apikey()
 @require_GET
-def heartbeat(request, domain, id):
-    # mobile needs this. This needs to be revisited to actually work dynamically (Sravan June 7, 17)
-    for_app_id = request.GET.get('app_id', '')
-    return JsonResponse({
-        "app_id": for_app_id,
-        "latest_apk_version": {},
-        "latest_ccz_version": {}
-    })
+def heartbeat(request, domain, app_id):
+    """
+    An endpoint for CommCare mobile to get latest CommCare APK and app version
+        info. (Should serve from cache as it's going to be busy view)
+
+    'app_id' (that comes from URL) can be id of any version of the app
+    'app_id' (urlparam) is usually id of an app that is not a copy
```

Comment:
Sure. FWIW, both of them come from mobile. Mobile gets `hq_app_id` from the resource URL and  urlparam app_id from other resource xml files